stable/field-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "field-exporter.fullname" . }}-controller-manager
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: field-exporter
+    app.kubernetes.io/part-of: field-exporter
+    control-plane: controller-manager
```

Comment:
What is this label for?

Reply:
@arjunrn or @mtahaahmed can you answer this question?

Reply:
This helm chart was converted from kustomize. I wanted to change in the helm chart after I see it running in the cluster.

Reply:
we can probably remove it for now